spec/octokit/client/apps_spec.rb

```diff
@@ -16,7 +16,7 @@
     it "returns installations for an integration" do
       installations = @jwt_client.find_integration_installations
       expect(installations).to be_kind_of Array
-      assert_requested :get, github_url("/integration/installations")
```

Comment:
I'd really like to keep the old tests around so that when someone removes the aliases the tests notice.

I'm not too worried about duplication or anything like that.